provider/datagen/src/transform/icuexport/collator/test.rs

```diff
@@ -205,3 +206,65 @@ fn test_nb_nn_no() {
         Some(&locale!("no").into())
     );
 }
+
+#[test]
+fn test_zh() {
+    let provider = crate::DatagenProvider::for_test();
+    let provider = LocaleFallbackProvider::try_new_unstable(provider).unwrap();
+    let provider = ResolvedLocaleAdapter::new(provider);
+
+    let input_locale = locale!("zh-u-co-gb2312").into();
+    Collator::try_new_unstable(&provider, &input_locale, CollatorOptions::new()).unwrap();
```

Comment:
Why doesn't this fail given that `gb2312` isn't explicitly included?

Reply:
Yeah, so if you specifically request it and the data is there, it will load. I'm only removing it from the `IterableDataProvider` impl, so that when you _list_ the supported collations, you don't get it.